src/boot/efi/UEFI_SECURITY.md

```diff
@@ -4,6 +4,13 @@ PE binary, adding various features, `systemd-stub`. These components fully suppo
 this document will describe their security posture and how they comply with industry-standard expectations
 for UEFI SecureBoot workflows.
 
+Note that `systemd-stub` is not the same, or an alternative, to the Linux kernel's own EFI stub. The kernel
+stub's role is to prepare to launch the 'proper' kernel execution from EFI. `systemd-stub` on the other hand
```

Comment:
I think we should probably use the words "entrypoint" and "boot protocol" here.

Maybe:

The kernel stub's role is that of the fundamental entrypoint to kernel execution from UEFI mode, implementing the modern Linux boot protocol.